docs/organizations/managing-security-and-risk.md

```diff
@@ -131,6 +131,40 @@ The following section details when Codacy opens and closes items for each suppor
         -   If an issue is opened and closed on the same day, Codacy may not detect it.
         -   To make sure that Codacy detects Jira issues correctly, assign the "security" label when creating the issue or immediately after.
 
+## Managing access to security and risk management
```

Comment:
Does it make sense to move the **Managing access to security and risk management** section right after the **Managing integrations** instead? 🤔 

This way we would gather the two procedures that users can do on the screen, and keep reference information below the procedures.

Also, the **Opening and closing items** section looks like reference information, because they're not actions that users can perform. A couple of notes on this:
-   We are naming procedure sections with a gerund, such as Managing integrations, Installing/Uninstalling Jira integration, and so on. Thus, maybe we should rename this section to something not gerund, to avoid the expectation of a user procedure. Eg, **Open and close items**.
-   Is the content in **Open and close items** related to "Item statuses"? Does it make sense to gather these two in the same section? 🤔 

Thoughts?